src/components/Button/Button.module.scss

```
@@ -2,17 +2,22 @@
 @use 'styles/mixins.module.scss' as mxn;
 
 .button {
+  cursor: pointer;
   position: relative;
-  padding: 8px 14px;
+  padding-top: auto;
+  padding-left: 14px;
+  padding-right: 14px;

```

будто короче и не менее понятно будет так:
```suggestion
  padding: auto 14px 0;
```